frontend/src/components/Header/UserInfo.js

```diff
@@ -1,9 +1,12 @@
-import React from 'react';
+import React  from 'react';
```

Comment:
Remove extra space between "React" and "from."

Reply:
Fixed.